recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = "fluidsim" %}
-{% set version = "0.3.1" %}
-{% set build = 0000 %}
```

Comment:
This is probably a bug in the bot, right @CJ-Wright?

Reply:
Kinda? The bot looks for `build` in the recipe since this is usually the way to set the build number, but I don't think it knows the difference between 0000 and 0 since it treats them as ints. Is there a difference here?